database/seed/seedS3.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-await-in-loop */
+const { uploadRandomImage } = require('../../utils/s3bucket');
+const { rng } = require('../../utils/math');
+
+const maxProducts = 1000;
+const maxStyles = 3;
+const maxOthers = 8;
+
+(async function loop() {
```

Comment:
Loop is too generic of a name; I highly recommend re-naming this method to something more clear for clarity. I.e uploadToAWS, awsSeed, S3seed or anything that helps you and the reviewer know it’s for the bucket. Another good example is the ‘uploadRandomImage’ is a good example too; I as the reviewer know exactly what the method does.